### DISCUSSION TITLE
Want to show a menu only on the frontpage

### DISCUSSION THREAD

**Discussion by PH5HP** [e107inc/e107#5289] [General]
Hi,

I have an issue to display a menu only on the FRONTPAGE. If I set the filter then it works just one time and if I go back from another page to the main page my menu with the filter to be displayed only on the frontpage does not show on the frontpage any more.

I tried FRONTPAGE and even /!. Both gives me the same result. I can see it on the frontpage the first time after I set the visibility. After that I go to another page and when I go back to the frontpge it does not show anymore. This happens every time.

Anyone? I am using BS GAMING 01 as my theme (just for info).

Kind regards,
Henk - ph5hp.nl

**Comment by Jimmi08** [comment]
My guess is that going back to your URL is different. Like www.site.com and www.site.com/index.php 

What is your frontpage? news.php or index.php?    

I tested this with news.php

FRONTPAGE  is site.xy  - it works
site/news.php - it is not frontpage, it is not showed, it is works
site/index.php - it is not frontpage, it is not showed, it is works  

So check your links in navigation.
If you consider index.php as frontpage, you need to add it into visibility filter.

**Comment by Moc** [comment]
Please also check if your "Site URL" is set correctly in the `Admin Area > Preferences > Site Information` section. 

In the menu manager visibility section, make sure you set one page per line.

**Comment by PH5HP** [comment]
Hi,

Ok ... I thought it was something like this. I added index.php and FRONTPAGE.  FRONTPAGE works if indeed if no news.php is in the URL and if I go from that to another page and back to the frontpage then news.php was added. So that's why I added both and it seems to work now.

Thank you for al your help.

Kind regards,
Henk - ph5hp.nl
